pom.xml

```diff
@@ -105,7 +105,7 @@
   <dependencies>
      <dependency>
       <groupId>org.openrefine</groupId>
-      <artifactId>main</artifactId>
+      <artifactId>core</artifactId>
       <version>${openrefine.version}</version>
       <scope>provided</scope>
```

Comment:
If changing from `main` to `core` keeps the build happy, then it's worth doing, as it will ensure your extension relies on the more stable classes in OpenRefine. If it doesn't, you can still keep `main` here.